pkg/apis/v1/types.go

```diff
@@ -614,29 +598,27 @@ func (sh *strictHandler) GetProbeById(w http.ResponseWriter, r *http.Request, cl
-	"gTHOvYLbErSxqaVkAcpwsOyzgjtSP2PvHxBVh1RW3mayPDVAvD0qf764vToZnh8ITTp+asdmu1neXotW",
-	"EG5qVuT0K8QG4f2mlFRXoAspNOzyAfh693EsE9jV0NuPH8dEG2ZKTXBESxZREGzIlQtzMGj0yoWBGSjL",
-	"AGjNZh2rn5J5mTPhK2AJm2ZALDayHt92z3uuNRczUrWO1GfjkHzpbNMXIhX5svF710zbVGAPGry7zd0a",
-	"71r5IAcfpHktS2FFx7LsMqXDT/dLp03hynsihxsMRe3+N/20piWJBC3+/vMvQ+A711gh1rhVzk33Efbc",
-	"qGyWIucPp+YxBCw4imI/DeHAj04Oj/wpHB360eAQXoVhGA0gekxq2novXUU7kK+gUKCRAsIIyiyrEjGW",
-	"IuWzUjEc2bPOe+Fw+0+G2ouFmaVNnyrFlvsjzVKlO/IFpyGFhnGB4SEFYCjkUlX8up20vQQayPVD9W/K",
-	"qHEKw412mrCG1REjSLtIZQfi8cjee2yrEPFZxuI/pvI7cY2wL68uJ6fWSefVrRYFz431y9Xby7MJmSyF",
-	"mYPhsa4mno5H1KMLUNrtFPSCXoglyAIEKzgd0oNe0DtAPpiZ20b0c/xgiHW/afAMrH2w+9YSo4QO6Tuu",
-	"jdvGzm6+WfbEXDOkf//VGJPvgQXu+TxY3SAfTjQW/CAIXEwKA8I4/xYZXpK4FP2vGhtz98jrb5cwLbHb",
-	"R1zGtbHpl2V1muDxZef3kIHoBWFtnRy7gEZiwTKe2HMUtGmOUcSC9/Yyz5la0iF9A4aw++Gj7thMb4od",
-	"o0TqDpFs3BXX34egzZlMli9We8dtdNX2JH6LrnZEEf5sUbizN7ZwE6LLOAat0zLLltscuJLwVBLwzTW9",
-	"q+crb9ur/bsmYlcuZzJw8d6m5cI+r2h5mnk7/pDo8Fy0G3Pry4fdeqsB5IMkay7W5oh+jDnqW9legoQ0",
-	"JMUh26y4niEr7hDJmYnnmNaFkgueQEJGF93W6IzPN+DS82w5Sn4QBz899863oqLp0fprs+rTv5ZkF38O",
-	"/HRZX1W7mbW3ZvdwW+uXFdeaKMis440ka7c2OVr/62R/r25W/wQAAP//a7itHgwUAAA=",
+	"H4sIAAAAAAAC/8xX247bNhD9FYLtQwvIt8S5GcjDbrZIDARddzf7FAQLWhxZTChSS46cuIb+vRhS8k3a",
+	"JinSbR8tjckzZ86ZGW15aovSGjDo+WzLS+FEAQgu/HorlqCvQUOK1v1Rgdss6D29kuBTp0pU1vAZP2Op",
+	"LQox8EAHIEimlUdmM/YJNi/XQlfANB3mGVqWKY3gmDVDnnD4IopSA5/xVFcewd0q+VI+ejHOJgCDp+mT",
+	"6WC6HE8GL8bwdCCfjSfPps+z8fMnk6R0ai0QXqKrgCdcEZA7AskTbkRBR4Y7b32TAU+4T3MoBCWAm5Ii",
+	"PDplVryuE75wdglzuRCY35PmuxzY/ILSwhxYSfGUjwN0CtZwnM635NDCLgXme9Th4FslecId3FXKgeQz",
+	"yvIQ/88OMj7jP432BRzFt37UZHIdg2tKrnlF/3zlQCCEmCu4q8BjqLyzJThUEGJisb52T9CHb69JuEeB",
+	"Kr2tnP7aP69D5I3TO4yHub4/POlD0tbKLj9CinTRb85ZdwW+tMZDFz3Q6+7j1EroFvXNu3cLRvdVnlHE",
+	"URWn43HCM+sKgXzGlcHHj/gOjzIIK3AEqADvxQr6nJFXhTADB0KKpQYWsLEm/lgxc7MWWsnoFNaqljXX",
+	"Jz2SPSGm7qHqqEazLRdSKsIm9OKInJPDk04eHlpDD6KhS6GcZ5gLZKkwbAms8iDJENathFF/AhNGNnlE",
+	"t/ijhLcHlv92wzS25zMejN+X87H+e21cGXVXAVMSDKpMUTvKmGg8/cvNzfyi4f3Xf+TqnWSqKvi4w26A",
+	"eBkRdwBeQenAk2GYYF6ZlW67TWpNplaVExQ5DGwcFpF4/K7WkPznTg/0fMXuAbQ/c05s7jd9FFiPA+lv",
+	"RBwKZZRZMWuAkQWta1mNNwVxKoTCfxOJlzt4DV5BF3Xya2D1pXVKTa9QIzXs5uotOWvZQJbHqswRSz8b",
+	"jUSphs3TQeOtYWbtUMLa5yrDoXWrI3U63dNXappKme2hcjEnV7BCGLEiKs+1SD8t7RcWK0RnKQyIrt5c",
+	"nl+z643BHFClvolgZ4s5T/ganI9Hjofj4YTIsCUYUSo+44+Hk2EwusA8lGJU0IBN/Whf4hUE21D9gxXm",
+	"1EDeKo87IIebzPv+cu5DRvdtOvUHqmaUXLj40XgcR4lBMAGDKEut0oBi9NFTUtvvGdMnsg7sn7bedpkS",
+	"Wu86AMhdS60TPv2BsI6Haw+gdk65uD2wPY/DIH9fFYVwGz7jrwGZ+Hv4pBmx8odWqRNeWt9T4IPNpdmN",
+	"wOO5lZsflnvPblQfO5qGTt0RxeTHiqLtLV3uF3EMBJiS+SpNwfus0nrT6ODFw+ngrGmfnxXmYSP2omgb",
+	"FvVyJjStPRsGX5THKNQnDytUBGcE7VJuDS7uXqcajSWnSWvgc8yoT5N1ctqHRtt2Ua9jq9SA0BXtRXje",
+	"ivb72lLnk6SnH027fTqKJAI6EQn73bKG/EYw04erR8RlLLLMVkaeViIy5XdrWCEwzWnMlM6ulQTJ5hf9",
+	"7aJ3HLyGOA3ON3P5rzA/fijTvzppmntmmu/Qlp3/XUnjAIiglxum0N9Tw/DtEh+eqvmyrapnDnToe2hZ",
+	"48X9FDn8ePa8/lD/FQAA//8Nf/Gi3BAAAA==",
```

Comment:
I'm not familiar with Swagger: is it normal to include encoded data like this in it's files?

Reply:
It's autogenerated, but from what I've learned that will get used to:
* Expose the API Documentation GUI (not done yet, but plan on it)
* Validate incoming and outgoing requests against the defined api contracts 
https://github.com/rhobs/rhobs-synthetics-api/blob/main/cmd/api/main.go#L21